### PR TITLE
chore: release automations

### LINKED ISSUE #12: Make tagged releases
Hey, thanks for this skill!

I've been packaging Mycroft for Alpine Linux (which I think is the first distribution to do so) and am currently packaging all the different skills available. The idea is to make all skills available as distro packages and not use the current Mycroft marketplace and Pip to install them. I'd love to package your skill as well, but will need a tagged (`git tag`) release to do so.

Please consider making tagged release so that distributions like Alpine Linux and in the future others can package them and keep track of updates.

### LINKED ISSUE #20: Not on PyPi

### PR DESCRIPTION
**Pull request by mikejgray**
Closes #20
Closes #12 

## Summary by CodeRabbit

- **New Features**
	- Introduced new workflows for automated build testing, release processes, and pull request labeling based on conventional commit messages.
  
- **Bug Fixes**
	- Removed outdated workflows that were no longer necessary, streamlining the CI/CD process.

- **Documentation**
	- Updated README and skill.json to reflect changes in settings configuration formatting.

- **Chores**
	- Added new testing dependencies to enhance the development environment.
	- Updated workflow configurations to improve compatibility and performance.

### CODE REVIEW

File: .github/workflows/build_tests.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      - dev
+  workflow_dispatch:
+
+jobs:
+  build_tests:
+    strategy:
+      max-parallel: 2
+      matrix:
+        python-version: ["3.10", "3.11", "3.12", "3.13"]
+    runs-on: ubuntu-latest
```
[the remark this answers is not in the document]
Comment (mikejgray) [reply .github/workflows/build_tests.yml:17]: Silly bot, Python 3.13 is GA now!

File: .github/workflows/sync_tx.yml [hunk 2]
```diff
@@ -11,27 +11,22 @@ jobs:
...
           ref: dev
           fetch-depth: 0 # otherwise, there would be errors pushing refs to the destination repository.
       - name: Setup Python
-        uses: actions/setup-python@v1
+        uses: actions/setup-python@v5
         with:
-          python-version: 3.9
-          
-      - name: Run script if manual dispatch
-        if: github.event_name == 'workflow_dispatch'
```
Comment (JarbasAl) [inline .github/workflows/sync_tx.yml:24]: this one should stay, it is useful when translation sync fails  (eg, network issues)

File: .github/workflows/sync_tx.yml [hunk 3]
```diff
@@ -11,27 +11,22 @@ jobs:
...
-        run: |
-          python scripts/sync_translations.py
```
Comment (JarbasAl) [inline .github/workflows/sync_tx.yml:26]: script missing?

Comment (mikejgray) [reply .github/workflows/sync_tx.yml:26]: No, it's in the scripts folder on this branch

**Review by JarbasAl — APPROVED** [review]

**Pull request merged**